src/cryptsetup/cryptsetup-tokens/cryptsetup-token-systemd-empty.c

```diff
@@ -0,0 +1,74 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+
+#include <errno.h>
+#include <libcryptsetup.h>
+
+#include "cryptsetup-token.h"
+#include "cryptsetup-token-util.h"
+#include "memory-util.h"
+#include "version.h"
+
+#define TOKEN_NAME "systemd-empty"
```

Comment:
A naive question: TOKEN_NAME is not used anywhere. How does cryptsetup know to only use this code when "systemd-empty" token is enrolled?

Reply:
Maybe even better to rename that define to reflect the actual usage of it, it will make it more clear when sources are inspected by others in future :wink: 
eg: `TOKEN_NAME` -> `SYSTEMD_EMPTY_TOKEN` or similar...

Reply:
Whenever cryptsetup needs to open a token with type set to "foobar", it'll dlopen /usr/lib/cryptsetup/libcryptsetup-token-foobar.so and call the functions in there. The files src/cryptsetup/cryptsetup-tokens/cryptsetup-token-*.c each get linked into their own .so file

Reply:
TOKEN_NAME was left over from a copy+paste error. In the other tokens, it's used when putting together error messages. I've dropped it